core/services/chainlink/application.go

```diff
@@ -378,6 +378,7 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 				globalLogger,
 				registry,
 				legacyEVMChains,
+				cfg.Capabilities().Peering().PeerID().String(),
```

Comment:
This usually won't be in the config since Peer can just fetch it from Keystore. Can we extract it from PeerWrapper?